<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CInfoBarWebReadersStatusComponent</class>
 <widget class="QFrame" name="CInfoBarWebReadersStatusComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>22</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Web readers status bar</string>
  </property>
  <layout class="QHBoxLayout" name="hl_InoBarWerbReaderStatusComponent">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="lbl_DataReady">
     <property name="toolTip">
      <string>Mappings ready</string>
     </property>
     <property name="text">
      <string>data</string>
     </property>
     <property name="indent">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="swift::gui::CLedWidget" name="led_DataReady" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="lbl_Consolidation">
     <property name="toolTip">
      <string>consolidation</string>
     </property>
     <property name="text">
      <string>cons.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="swift::gui::CLedWidget" name="led_Consolidation" native="true"/>
   </item>
   <item>
    <spacer name="hs_Spacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="lbl_Loading">
     <property name="text">
      <string>loading:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_Models">
     <property name="text">
      <string>models</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="swift::gui::CLedWidget" name="led_Models" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="lbl_Liveries">
     <property name="text">
      <string>| liveries</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="swift::gui::CLedWidget" name="led_Liveries" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="lbl_Distributors">
     <property name="text">
      <string>| distributors</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="swift::gui::CLedWidget" name="led_Distributors" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="lbl_Aircraft">
     <property name="text">
      <string>| aircraft</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="swift::gui::CLedWidget" name="led_IcaoAircraft" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="lbl_Airlines">
     <property name="text">
      <string>| airlines</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="swift::gui::CLedWidget" name="led_IcaoAirline" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="lbl_Countries">
     <property name="text">
      <string>| countries</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="swift::gui::CLedWidget" name="led_Countries" native="true"/>
   </item>
   <item>
    <spacer name="hs_InfoBarWebReadersStatusComponent">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::CLedWidget</class>
   <extends>QWidget</extends>
   <header>gui/led.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
